pkg/controllers/controllers_test.go

```diff
@@ -19,26 +19,28 @@ import (
 
 	"code.cloudfoundry.org/cf-operator/pkg/credsgen"
 	gfakes "code.cloudfoundry.org/cf-operator/pkg/credsgen/fakes"
+	"code.cloudfoundry.org/cf-operator/pkg/kube/util/config"
+	"code.cloudfoundry.org/cf-operator/testing"
 	"github.com/SUSE/eirini-extensions/pkg/controllers"
 	cfakes "github.com/SUSE/eirini-extensions/pkg/controllers/fakes"
-	"github.com/SUSE/eirini-extensions/pkg/util/config"
-	"github.com/SUSE/eirini-extensions/testing"
 )
 
 var _ = Describe("Controllers", func() {
-	Describe("AddToScheme", func() {
-		It("registers our schemes with the operator", func() {
-			scheme := scheme.Scheme
-			controllers.AddToScheme(scheme)
-			kinds := []string{}
-			for k, _ := range scheme.AllKnownTypes() {
-				kinds = append(kinds, k.Kind)
-			}
-			Expect(kinds).To(ContainElement("BOSHDeployment"))
-			Expect(kinds).To(ContainElement("ExtendedSecret"))
-			Expect(kinds).To(ContainElement("ExtendedStatefulSet"))
-		})
-	})
+	// We don't have any controller right now
```

Comment:
Not sure what to do here - we will have them in the future?

Reply:
Yup, wanted to do it in a separate PR to address all the other areas that needs names refactoring, here I mostly focused to have the test suite running